### ISSUE TITLE
Enable navigation title by default for all content objects

### ISSUE THREAD

**Issue by tisto** [plone/plone.volto#177]
Having the option to choose a different navigation title is essential for a CMS. When you have long page titles, you have to shorten them for the navigation and the breadcrumbs. Therefore this option should be present by default in Plone.

cc @sneridagh

**Comment by davisagli** [comment]
@tisto The option is present: we have the navtitle behavior, which can be enabled. Did you mean that it should be enabled by default for all content types in a new site?

**Comment by tisto** [comment]
@davisagli yes. The navtitle behavior should be enabled by default IMO. We used to have a folder and a default view that can have different titles in Plone. This option is gone since we moved to folderish pages. Therefore we have to enable the navtitle behavior by default to have the same functionality.

**Comment by mauritsvanrees** [comment]
I am removing this from the [Plone 6.1 project board](https://github.com/orgs/plone/projects/25).  Plone 6.1.0 has been released, so the board should be closed.  It seems small enough to be done in a bugfix release of Plone 6.1, but it does not need to be on a board for that.

**Comment by davisagli** [comment]
@mauritsvanrees That's fine to remove it from the Plone 6.1 board. This is a breaking change but in this case that means we'll make the change without adding an upgrade step, so that it only applies to new sites. So it can happen whenever.

**Comment by sneridagh** [comment]
I just added this to 6.2 board.
